src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -1,43 +1,43 @@
 ﻿using System;
-using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using NHSD.BuyingCatalogue.Ordering.Api.Models;
+using NHSD.BuyingCatalogue.Ordering.Application.Persistence;
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.Controllers
 {
-    [Route("api/v1/[controller]")]
+    [Route("api/v1/organisations/{organisationId}/[controller]")]
```

Comment:
It might be better to add this specific route defined here to the `GetAllAsync` method instead & change this route back to the `[Route("api/v1/[controller]")]`. This way when changes get merged by other members of the team, the base route path won't change as the common path for orders is `api/v1/orders`.

Reply:
Due to the route being different, I can add this to be common: [Route("api/v1")], but as the organisations route is different I've had to do this
[HttpGet]
        [Route("organisations/{organisationId}/[controller]")]

Reply:
One suggestion is to prefix your specific organisations route with a `/` this should discard everything before it. Comments made [here](https://github.com/aspnet/Routing/issues/320)

Reply:
👍